src/languages.ts

```diff
@@ -37,6 +37,13 @@ export function parseLanguage(language: string): Language | undefined {
 }
 
 export function isTracedLanguage(language: Language): boolean {
+  if (process.env["CODEQL_EXTRACTOR_GO_BUILD_TRACING"] === "true") {
+    throw new Error(
+      "The CODEQL_EXTRACTOR_GO_BUILD_TRACING environment variable is set to an invalid value. " +
+        "To enable Go build tracing, please set it to 'on'."
+    );
+  }
```

Comment:
Alternatively, instead of throwing an error, we could set the env var to `on`.

```suggestion
  if (process.env["CODEQL_EXTRACTOR_GO_BUILD_TRACING"] === "true") {
    process.env["CODEQL_EXTRACTOR_GO_BUILD_TRACING"] = "on";
  }
```

Maybe add a warning, too.

Reply:
Good point, let's do this.  We'll also need to make a `core.exportVariable` call to update the value of the environment variable in subsequent steps.